dynawo/3rdParty/omcUpdate_1_9_4/omcUpdateDynawo.sh

```diff
@@ -152,7 +151,7 @@ architecture_omc() {
   MAKEFILE_LIBS="Makefile.libs"
   SIMULATION_DATA_FILE="simulation_data.h"
   TASK_GRAPH_RESULTS_CMP_FILE="TaskGraphResultsCmp.cpp"
-
+  MAKEFILE_IN="Makefile.in"
```

Comment:
I don't see this file in the patch folder?

Reply:
The patch file was forgotten in the commit !